### PR TITLE
Theme/#408 remove inline styles

### LINKED ISSUE #408: Remove Inline Styles from our ACF Blocks
A general plan of attack:
- Stop using the ACF color picker and instead use a filtered & styled select dropdown with defined colors. We already define colors and hide the color picker, so why not just go the next step and stop using the color picker all together? This would also eliminate the need for admin JS.
- This allows us to set class names for colors (`background-green`, `color-green`, etc.) and use those instead of inline styles
- Videos are already not inline, so I think we're good there
- Instead of using images as inline background styles, use a `<figure>` element to place them inside of the container and position the same way we're doing with "background" videos

This would get us away from adding inline styles to our blocks at all, as everything would either be handled by a class (background colors, text colors) or inline elements positioned as background elements (videos, images).

### PR DESCRIPTION
**Pull request by coreymcollins**
Closes #408 

### DESCRIPTION ###
This replaces all of our inline styles and, in doing so, gets us more compliant with Section 508 and WCAG 2.0 AA. Neat!

### Color Picker
We went to great lengths in the past to customize the ACF Color Picker field to only show the brand/theme colors we specified while hiding the color picker gradient itself. While this made sense in a general sense, it seemed like kind of overkill to use the Color Picker field if we're just going to hide the actual picker.

This removes our usage of the Color Picker from Background and Other Options in favor of a standard `select` field which we filter via a PHP function in the theme's files.

Instead of adding these values as an inline style, they now get added as classes like `background-blue` for background colors and `text-cod-gray` for text colors.

### Image Background
The image background was the other item still being used as an inline style. This update moves away from that and displays the image as a `<figure>` in the same way we're displaying Video Backgrounds with positioning and z-indexing to place it behind the content while centering it vertically and horizontally.

### Video Background
No changes here. I've combined some styles with new styles for the image background, but have not added any new styles for functionality for the video background.

### Global Content Block Renaming
I renamed the "Block Options" field group to "Other Options". The tab itself on our blocks is "Other Options", so I renamed the field group to match.

### Total Validator
This gets rid of all of the dings against our use of inline styles. Any remaining warnings or issues have to do with empty `alt` tags and TVP disliking WP's use of the `sizes` attribute on `img` tags within these blocks.

### SCREENSHOTS ###
**Desktop**

**Mobile**

### OTHER ###
- [x] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [x] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)
- [x] Does this pass CBT?

### STEPS TO VERIFY ###
Check out the branch and add any combination of blocks, testing the usage of background images, background colors, and background videos as well as text colors.

### DOCUMENTATION ###
Yes – we'll need to update a screenshot on this page if this is accepted: https://github.com/WebDevStudios/wd_s/wiki/ACF-Content-Blocks#other-options

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -263,3 +263,28 @@ function _s_copyright_year( $atts ) {
...
+
+/**
+ * Get the theme colors for this project. Set these first in the Sass partial then migrate them over here.
+ *
+ * @return array The array of our color names and hex values.
+ * @author Corey Collins
+ */
+function _s_get_theme_colors() {
+	return array(
+		__( 'Alto', '_s' )           => '#ddd',
```
Comment (gregrickaby) [inline inc/extras.php]: @coreymcollins The translation functions should be 

```suggestion
		esc_html__( 'Alto', '_s' )           => '#ddd',
```

**Review by gregrickaby — CHANGES_REQUESTED** [review]

**Comment by coreymcollins** [comment]
Pushed up a change to address the translations. Should be set there!

**Comment by gregrickaby** [comment]
@coreymcollins Can you fix up the merge conflicts quick? Thank you!

**Comment by coreymcollins** [comment]
Should be good to go now – merged `master` in and recompiled styles.

**Review by gregrickaby — APPROVED** [review]

**Pull request merged**